packages/rollup-plugin/__tests__/index-test.js

```diff
@@ -62,27 +62,64 @@ describe('rollup-plugin-stylex', () => {
     const { css, js } = await runStylex({ fileName: 'stylex.css' });
 
     expect(css).toMatchInlineSnapshot(`
-      "
-      @layer priority1, priority2, priority3, priority4;
-      @layer priority1{
-      @keyframes xgnty7z-B{0%{opacity:.25;}100%{opacity:1;}}
+      "@layer priority1 {
```

Comment:
Look into the options for lightningcss so we format CSS in a more minimal formay.

Reply:
I couldn't find such an option, so maybe it's OK to leave it expanded.

Reply:
We can use `minify: true` option for this.
For example:
```
import { transform } from 'lightningcss';

let { code, map } = transform({
  filename: 'style.css',
  code: Buffer.from('.foo { color: red }'),
  minify: true,
  sourceMap: true
});
```

Documentation: https://lightningcss.dev/docs.html

Reply:
@Prakshal-Jain `minify: true` is more aggressive than we want. It will also get rid of all the line breaks and make the CSS generally unreadable. It should not be turned on by default.

However, the `rollup-plugin` should be updated accept props that are passed into `lightningcss` and it should be possible to pass in `minify: true` within `lightningcssArgs: {...}` by the developer.

Reply:
Okay. Let me disable this for now. I'll add the lightningcssArgs in the next commit to add support for this argument.

Reply:
Resolved. Back to original format.